src/inputs/MultiSelectField.test.tsx

```diff
@@ -48,9 +48,11 @@ describe("MultiSelectFieldTest", () => {
     expect(r.age()).toHaveValue("");
 
     // When blurring the field
-    fireEvent.blur(r.age());
+    // fireEvent.blur(r.age());
+    act(() => r.age().blur());
```

Comment:
Fwiw if we want to make our rtl-utils `blur` do a `.blur()` instead of `fireEvent.blur` that sgtm.

Reply:
I forgot about this change here. I revisited this and the failure seemed to be due to call `fireEvent.blur` twice. I took off the second `fireEvent.blur` call and it passes. I do find it odd that `focus` seems to really need to be wrapped in an `act`, while `blur` doesn't seem as strict about it.